### PR TITLE
Builder: Remove deprecated `Brocfile.js` support

### PR DESCRIPTION
**Pull request by Turbo87**
... and the now unused `broccoli-brocfile-loader` dependency

### CODE REVIEW

File: tests/acceptance/brocfile-smoke-test-slow.js [hunk 1]
```diff
@@ -1,427 +0,0 @@
-'use strict';
```
Comment (rwjblue) [inline tests/acceptance/brocfile-smoke-test-slow.js]: This file should not be removed. Even though it is named “brocfile” it is about ember-cli-build.js mostly (you can see that if you review the deleted fixtures)...

**Comment by stefanpenner** [comment]
Hmmm. I still use this, but I guess I can simply rename my brocfile’s

SG

File: tests/acceptance/brocfile-smoke-test-slow.js (hunk 1 above)
Comment (Turbo87) [reply tests/acceptance/brocfile-smoke-test-slow.js]: @rwjblue fixed

**Comment by Turbo87** [comment]
@rwjblue any news about this?

**Comment by twokul** [comment]
related to https://github.com/ember-cli/ember-cli/issues/7252

**Comment by Turbo87** [comment]
@rwjblue friendly reminder :)

**Pull request merged**